tests/Microsoft.DotNet.Framework.Docker.Tests/ImageTests.cs

```diff
@@ -321,7 +321,7 @@ private static string GetManifestRegistry()
 
         private void VerifyHttpResponseFromContainer(string containerName, string urlPath)
         {
-            var retries = 30;
+            var retries = 60;
```

Comment:
This change can be reverted.  The original retry count of 30 works fine when the Dockerfile change is applied.